integration/use-cases/add-multicluster-deployment.js

```diff
@@ -0,0 +1,35 @@
+const utils = require("./lib/utils");
+
+test("Deploys an application with the values by default", async () => {
+  await utils.login(
+    page,
+    process.env.USE_MULTICLUSTER_OIDC_ENV,
+    "/",
+    process.env.ADMIN_TOKEN,
```

Comment:
I would not add this `process.env.ADMIN_TOKEN` (just to make clear this test won't work with a token)